src/Domain.h

```diff
@@ -382,7 +356,6 @@ class Domain {
 	bool thermostatWarning() { return (this->_universalSelectiveThermostatWarning > 0); }
 
 	void evaluateRho(unsigned long localN, DomainDecompBase* comm);
-        void submitDU(unsigned cid, double DU, double* r);
```

Comment:
This function, submitDU, was related to the outputs, right?

Edit: restore the function signature and call, maybe comment-out the function implementation body. See below.